test/IRGen/objc.swift

```diff
@@ -139,7 +139,7 @@ class WeakObjC {
 // CHECK:  i32 1, !"Objective-C Version", i32 2}
 // CHECK:  i32 1, !"Objective-C Image Info Version", i32 0}
 // CHECK:  i32 1, !"Objective-C Image Info Section", !"__DATA,__objc_imageinfo,regular,no_dead_strip"}
-//   100665088 == (6 << 24) | (0 << 16) | (7 << 8).
+//   100730624 == (6 << 24) | (1 << 16) | (7 << 8).
 //     5 and 8 is the current major.minor version. 7 is the Swift ABI version.
```

Comment:
Should we also update this comment to reference 6 and 1 as the current major.minor versions instead of 5 and 8 so that it makes sense in context?

Reply:
This was already fixed in https://github.com/swiftlang/swift/pull/76017/commits/3c204cc683dec633c3e344fa045e54a2598a060e